pyroute2/ipdb/interface.py

```diff
@@ -779,9 +779,18 @@ def error(x):
             #
             # One simple way to work that around is to remove
             # secondaries first.
-            rip = sorted(removed['ipaddr'],
-                         key=lambda x: self['ipaddr'][x]['flags'],
```

Comment:
That's much better if 0xffff is invalid value for flags :)
Ср, 13 июля 2016 г. в 23:13, Peter V. Saveliev <notifications@github.com>:

> In pyroute2/ipdb/interface.py
> <https://github.com/svinota/pyroute2/pull/268#discussion_r70699626>:
>
> > @@ -779,9 +779,18 @@ def error(x):
> >              #
> >              # One simple way to work that around is to remove
> >              # secondaries first.
> > -            rip = sorted(removed['ipaddr'],
> > -                         key=lambda x: self['ipaddr'][x]['flags'],
>
> I merged it first, but then realized, that it could be solved in another
> way, as
>
> key=lambda x: self['ipaddr'][x]['flags'] or 0xffff
>
> What woud you say?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/svinota/pyroute2/pull/268/files/4aeaea52e54db2541c3a333722643d933b99f01f#r70699626>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALks0T51dEVplLAqEtARh3MSLbZy7bPks5qVUb2gaJpZM4JLqJ8>
> .
>